.vscode/settings.json

```diff
@@ -1,4 +1,6 @@
 {
   "editor.defaultFormatter": "biomejs.biome",
-  "editor.formatOnSave": false
+  "editor.formatOnSave": true,
+  "github.gitAuthentication": false,
```

Comment:
What's happening here?

Reply:
changed the VS Code settings so it would format on save and wouldn't try to to github auth on it's own (VS Code github auth can be super buggy).